hack/update-protobuf.sh

```diff
@@ -10,7 +10,7 @@ To skip protobuf generation, set \$PROTO_OPTIONAL."
   exit 1
 fi
 
-if [[ "$(protoc --version)" != "libprotoc 23."* ]]; then
+if [[ "$(protoc --version)" != "libprotoc 3."* ]]; then
```

Comment:
Why are you moving this backwards?

Reply:
oops..i need to revert this..changed it based on my system